### PR TITLE
[17.0][MIG] account_move_line_product

### PR DESCRIPTION
**Pull request by AnaisLForgeFlow**
Standard migration

@ForgeFlow

### CODE REVIEW

File: account_move_line_product/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# Copyright 2019 ForgeFlow S.L. (https://www.forgeflow.com)
```
Comment (GuillemCForgeFlow) [inline account_move_line_product/__manifest__.py]:
```suggestion
# Copyright 2019-2024 ForgeFlow S.L. (https://www.forgeflow.com)
```
perhaps we add the copyright here, as a minor improvement

**Review by GuillemCForgeFlow — CHANGES_REQUESTED** [review]
One good improvement to the module documentation would be to add a CONFIGURE.md file under the readme folder, explaining that the user needs to have `Technical / Show Accounting Features - Readonly` enabled in order to see the menu `Accounting / Accounting / Journals / Journal Items` menu and `Journal Items` page in the `account.move` form view.
What do you think? 😄

**Comment by rousseldenis** [comment]
@AnaisLForgeFlow 
Could you modify your commit message : 

image


As it is not a fix but the real migration one.

File: account_move_line_product/views/account_move_line_view.xml [hunk 2]
```diff
@@ -0,0 +1,48 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<!-- Copyright 2019 ForgeFlow S.L. (https://www.forgeflow.com)
+     License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl). -->
```
Comment (GuillemCForgeFlow) [inline account_move_line_product/views/account_move_line_view.xml]:
```suggestion
<!-- Copyright 2019-2024 ForgeFlow S.L. (https://www.forgeflow.com)
     License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl). -->
```

**Review by GuillemCForgeFlow — APPROVED** [review]
You can also change the copyright in the other file, just to have it synced.
Apart from that, the rest looks great, thank you for the CONFIGURE file 👍🏽

**Comment by rousseldenis** [comment]
@AnaisLForgeFlow Don't forget to sign CLA if not done yet.

**Review by celm1990 — APPROVED** [review]

**Review by StefanRijnhart — APPROVED** [review]

**Pull request merged**